### ISSUE TITLE
Chrome does not maximise well, mouse offsets, when tiling off and System Title Bar in use

### ISSUE THREAD

**Issue by timrichardson** [pop-os/shell#412]
**Distribution (run `cat /etc/os-release`):**
 pop!os 20.04, shell from dev_focal
 
 
**Related Application and/or Package Version (run `apt policy pop-shell` or provide the latest commit if building locally):**

chrome, with Settings -> Appearance -> Use System Title Bar and Borders enabled.

**Issue/Bug Description:**
When tiling is turned off, double clicking in the title bar resizes the window but with the wrong placement. 
On the middle of three monitors in horizontal layout, it seems to get the size correct, but it is shifted left and up by a few pixels (looks like the space used in tiling). However, consistent with a maximised window, it can't be moved. 
Also, mouse clicks are displaced ... mouse clicks link on where the window should be, not where the window actually is. This makes it very hard to click on the title bar to unmaximise.

**Steps to reproduce (if you know):**
This was tested on a machine with three external monitors.
Open Chrome. 
Settings -> Appearance -> Use System Title Bar and Borders enabled.
with tiling on, maximise, unmaximise

turn tiling off.
resize the chrome window to make it say about 2/3 of your screen. move it down to bottom left of the screen.
Double click title bar to maximise.
Now, try unmaximising

(reenable tiling to restore)

**Expected behavior:**

**Other Notes:**

**Comment by jmmaranan** [comment]
@timrichardson - I think this is caused by Snap to Grid setting (which is rarely used I believe). This is being used in the mode you were mentioning here in #461
![image](https://user-images.githubusercontent.com/348125/89072800-41e6b980-d347-11ea-8255-189cc0f7c394.png)

**Comment by timrichardson** [comment]
Agree, thanks, good catch. The grid is not very helpful, so I have turned it off, and it fixes the problem.
